src/mock/quic_p2p/tests.rs

```diff
@@ -692,17 +692,16 @@ fn gen_message() -> NetworkBytes {
             id::FullId,
             messages::{DirectMessage, Message, SignedDirectMessage},
         };
-        use lazy_static::lazy_static;
 
-        lazy_static! {
-            static ref FULL_ID: FullId = FullId::new();
+        thread_local! {
```

Comment:
any reason for this change?

Reply:
`FullId` is no longer `Sync` because it now holds `Rc` and `lazy_static` requires `Sync`.